docs/getting-started/devenv/nodejs-development-tools.md

```diff
@@ -0,0 +1,50 @@
+# Node.js development tools
+
+When developing for the Block Editor, you will need [Node.js](https://nodejs.org/en) development tools along with a code editor and a local WordPress environment. Node.js (`node`) is an open-source runtime environment that allows you to execute JavaScript code from the terminal (also known as a command-line interface, CLI, or shell)
+
+Installing `node` will automatically include the Node Package Manager (`npm`) and the Node Package eXecute (`npx`), two tools you will frequently use in block and plugin development.
+
+Node Package Manager ([`npm`](https://docs.npmjs.com/cli/v10/commands/npm)) serves multiple purposes, including dependency management and script execution. It's the recommended package manager and is extensively featured in all documentation.
+
+The Node Package eXecute ([`npx`](https://docs.npmjs.com/cli/v10/commands/npx)) tool is used to run commands from packages without installing them globally and is commonly used when scaffolding blocks with the [`create-block`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/) package.
+
+
+## Node.js installation on Mac and Linux (with `nvm`)
+
+For Mac and Linux, it's recommended that you use [Node Version Manager](https://github.com/nvm-sh/nvm) (`nvm`). This allows you to install and manage specific versions of `node`, which are installed locally in your home directory, avoiding any global permission issues.
```

Comment:
Maybe we can make this paragraph more generic to also invite Windows users to install nvm (as it's mentioned in the Windows section). What about something like the following?

> The recommended way to install Node.js is via [Node Version Manager](https://github.com/nvm-sh/nvm) (`nvm`). This allows you to [...]